### ISSUE TITLE
How to reuse an app while navigating between two navigation items which are using the same app

### ISSUE THREAD

**Issue by kistll** [SAP/luigi#1146]
**Description**

In our project, we use the LinkManager from LuigiClient to navigate. When we navigate to different navigation item the app which is provided via the viewUrl is always (re)loaded. The expectation would be that in case the app between the new and the old navigation item is the same, the app is not (re)loaded. E. g. if there is a navigation between the items 1-1 and 1-2, (re)loading makes sense as the navigation items are based on different apps (app1 and app2). If there is a navigation between items 1-1 and 1-3, there should be no (re)loading as both are based on the same app (app1).

```
{
    id: '1',
    pathSegment: 'root',
    viewUrl: null,
    children: [
	   {
		id: '1-1',
		pathSegment: 'first',
		viewUrl: '/app1/#/first',
	  },
	  {
		id: '1-2',
		pathSegment: 'second',
		viewUrl: '/app2/#/second',
	  },
	  {
		id: '1-3',
		pathSegment: 'third',
		viewUrl: '/app1/#/third',
	  }
    ]
}
```

**Expected result**

If there is a navigation between two navigation items which based app, the app should not be (re)loaded.

**Actual result**

If there is a navigation between two navigation items which based app, the app is (re)loaded.

**Comment by kistll** [comment]
Update: Is it mandatory to avoid (re)loading that those apps which are provided as viewURL make use of Luigi for navigation or can they use also a different kind of navigation?

**Comment by maxmarkus** [comment]
Mainly fixed with `v0.7.7`, but leave it open since there is still some reload going on in their special use case and it could be a bug on our side.
